example_app/dungeon.rb

```diff
@@ -101,12 +101,12 @@ def bottom
 class DungeonGenerator
   DEFAULT_WIDTH = 80
   DEFAULT_HEIGHT = 21
-  DEFAULT_TILESET_NAME = "default"
+  DEFAULT_TILESET_NAME = "default".freeze
```

Comment:
As before (and there's a couple more changes in this file and `example_app/game.rb` as well) - are these changes in scope for this PR or should they be separated?

Reply:
See above; agreed on separating out code style changes.